credits.js

```diff
@@ -0,0 +1,86 @@
+let Credits = {
```

Comment:
Your coding style is very neat and methodical. And the idea to add the cast and then link them to the series they appear in is a very good idea.

I noticed a small thing, when you click on any of the cast, it takes you to the page showing the list of series they appear in. But after that if you go to any other page, the series list stays at the bottom of the page even though we are no longer looking at that person's profile. Was that an intended feature or is it a bug?

Reply:
Definitely a bug! Thank you for spotting that, I'll fix it as soon as I can :)